spec/rails_helper.rb

```diff
@@ -10,35 +10,19 @@
 
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
-# require "rspec/rails"
-# Load any of our adapters and extensions early in the process
-require 'rspec/rails/adapters'
-require 'rspec/rails/extensions'
-
-# Load the rspec-rails parts
-require 'rspec/rails/view_rendering'
-require 'rspec/rails/matchers'
-require 'rspec/rails/fixture_support'
-require 'rspec/rails/file_fixture_support'
-require 'rspec/rails/fixture_file_upload_support'
-require 'rspec/rails/example'
-
-require 'rspec/rails/configuration'
-
+require "rspec/rails"
```

Comment:
@mfenner This requires ActiveRecord which Levreiro doesn't have. That is why I just load specific part of `rspec/rails`

Reply:
Thanks!